siphon/catalog.py

```diff
@@ -640,7 +729,7 @@ def __init__(self, service_node):
 
     def is_resolver(self):
         """Return whether the service is a resolver service."""
-        return self.service_type == 'Resolver'
+        return CaseInsensitiveStr(self.service_type) == 'Resolver'
```

Comment:
If you replace line 726 above with:
```python
self.service_type = CaseInsensitiveStr(service_node.attrib['serviceType'])
```
you could avoid doing the conversion here and at line 546 above.